client/state/ui/reducer.js

```diff
@@ -25,6 +26,31 @@ export function selectedSite( state = null, action ) {
 	return state;
 }
 
+export function section( state = false, action ) {
+	if ( action.type === SET_SECTION && action.section !== undefined ) {
```

Comment:
Do we need to check against `undefined` ? I worry that excessive validation hurts readability, when we should have the expectation that the actions be called with valid parameters.